ci/gpu/build.sh

```diff
@@ -35,42 +40,80 @@ export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
 # SETUP - Check environment
 ################################################################################
 
-logger "Get env..."
+logger "Check environment..."
 env
 
+logger "Check GPU usage..."
+nvidia-smi
+
 logger "Activate conda env..."
 source activate gdf
+conda list
+
+# Fixing Numpy version to avoid RuntimeWarning: numpy.ufunc size changed, may
+# indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
+conda install "cudatoolkit=$CUDA_REL" \
+              "cupy>=6.5.0" "numpy=1.16.4" \
+              "cudf>=0.10" "dask-cudf>=0.10" \
+              "dask>=2.3.0" "distributed>=2.3.2" \
+              -c rapidsai-nightly
+
+# needed for asynccontextmanager in py36
+conda install -c conda-forge "async_generator" "automake" "libtool" \
+                              "cmake" "automake" "autoconf" "cython" \
+                              "pytest" "pkg-config" "pytest-asyncio"
+conda list
+
+# Install the master version of dask and distributed
+logger "pip install git+https://github.com/dask/distributed.git --upgrade"
+pip install "git+https://github.com/dask/distributed.git" --upgrade
+logger "pip install git+https://github.com/dask/dask.git --upgrade"
+pip install "git+https://github.com/dask/dask.git" --upgrade
 
 logger "Check versions..."
 python --version
-gcc --version
-g++ --version
+$CC --version
+$CXX --version
+conda list
 
-# FIX Added to deal with Anancoda SSL verification issues during conda builds
-conda config --set ssl_verify False
+################################################################################
+# BUILD - Build ucx
+################################################################################
+
+logger "Build ucx"
+git clone https://github.com/openucx/ucx
+cd ucx
+ls
+./autogen.sh
+mkdir build
+cd build
+../configure --prefix=$CONDA_PREFIX --enable-debug --with-cuda=$CUDA_HOME --enable-mt CPPFLAGS="-I//$CUDA_HOME/include"
+make -j install
+cd $WORKSPACE
 
-conda install \
-    "cudf=$MINOR_VERSION.*" \
-    "dask-cudf=$MINOR_VERSION.*"
 
 ################################################################################
-# SETUP - Install additional packages
+# BUILD - Build ucx-py
+################################################################################
+
+logger "Build dask-cuda..."
+cd $WORKSPACE
+python -m pip install -e .
+
+
+################################################################################
+# TEST - Run py.tests for ucx-py
 ################################################################################
 
 if hasArg --skip-tests; then
     logger "Skipping Tests..."
 else
-    # Install CuPy for tests
-    pip install cupy-cuda${CUDA_REL}==6.0.0
-
-    # Install distributed@master (temporarily required due to issues with 2.3.2)
-    # pip install git+https://github.com/dask/distributed.git@master
+    logger "Check NICs"
+    awk 'END{print $1}' /etc/hosts
+    cat /etc/hosts
```

Comment:
Looks like this is not used anywhere, can we remove it?

Reply:
Check